LDAR_Sim/model_code/methods/deployment/orbit_crew.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, id, lat, lon, state, config, parameters, deployment_days, hom
         # obtain TLE file path
         m_name = self.config['label']
         self.sat = self.parameters['methods'][m_name]['satellite_name']
```

Comment:
self.parameters['methods'][m_name] can be replaced with self.config, they are the same thing. That's done elsewhere in the code occasionally but should be made consistent.

Reply:
fixed